plugins/geospatial/server/geospatial.py

```diff
@@ -263,15 +263,15 @@ def near(self, params):
                         raise ValueError
 
                 except ValueError:
-                    raise RestException("Parameter '{}' must be a number."
-                                        .format(param))
+                    raise RestException("Parameter '%s' must be a number." %
+                                        param)
 
-                condition['$'+param] = distance
+                condition['$%s' % param] = distance
```

Comment:
I'm not sure why I even changed this. My regex probably picked up something nearby, and I spotted it. I don't really care what this is.

Since you brought it up though, you're right it's faster with concatenation for 2 strings, but when you get to longer, non-trivial arrangements, formatting becomes much more maintainable and eventually faster. My thinking is to make everything use formatting, so future developers are encouraged to keep it that way if additional variables are necessary.